eng/scripts/Language-Settings.ps1

```diff
@@ -241,34 +253,34 @@ $PackageExclusions = @{
   '@azure/identity-cache-persistence' = 'Fails typedoc2fx execution https://github.com/Azure/azure-sdk-for-js/issues/16310';
 }
 
-function Update-javascript-DocsMsPackages($DocsRepoLocation, $DocsMetadata, $DocValidationImageId) {
+function Update-javascript-DocsMsPackages($DocsRepoLocation, $DocsMetadata, $DocValidationImageId, $SkipValidation=$false) {
```

Comment:
Do we need DocValidationImageId parameter any longer?

Reply:
The common script is calling this function, so it should be there until all languages moved to new validation tool